lib/config/options/index.ts

```diff
@@ -229,6 +229,17 @@ const options: RenovateOptions[] = [
     default: 'disabled',
     experimental: true,
   },
+  {
+    name: 'repositoryCacheType',
+    description:
+      'Set the type of renovate repository cache if repositoryCache is not disabled.',
+    globalOnly: true,
+    type: 'string',
+    allowedValues: ['local'],
```

Comment:
this won't work with `s3://...` values later

Reply:
i know https://github.com/Gabriel-Ladzaretti/renovate/commit/037a5bd4bf08dd6ae34e7c9cfc94d7738fcdc9ed

Reply:
we can discuss the s3 and redis whenever we are implementing it

Reply:
lets remove it here, when it's removed in next pr anyways